CONTRIBUTING.md

```diff
@@ -0,0 +1,7 @@
+# Contribution flow
+
+## Coding Conventions
```

Comment:
@githubteacher I do not like it when code comments are written in ALL CAPS. :blush: